cpp/include/kvikio/posix_io.hpp

```diff
@@ -131,15 +152,15 @@ ssize_t posix_host_io(int fd, const void* buf, size_t count, off_t offset, bool
 /**
  * @brief Read or write device memory to or from disk using POSIX
  *
- * @tparam IsReadOperation Whether the operation is a read or a write
+ * @tparam ioType Whether the operation is a read or a write.
  * @param fd File descriptor
  * @param devPtr_base Device pointer to read or write to.
  * @param size Number of bytes to read or write.
  * @param file_offset Byte offset to the start of the file.
  * @param devPtr_offset Byte offset to the start of the device pointer.
  * @return Number of bytes read or written.
  */
-template <bool IsReadOperation>
+template <IOType ioType>
```

Comment:
*nit*: I think one prefers template parameters to start with an uppercase letter. I think since every usage of the value is matched up with comparison with an `IOType::XXX` then we can get away with not writing `io` in the name as well.
```suggestion
template <IOType Type>
```

Reply:
Completed.